tensorflow/contrib/learn/python/learn/datasets/__init__.py

```diff
@@ -64,3 +73,34 @@ def load_dataset(name, size='small', test_with_fake_data=False):
     return DATASETS[name](size, test_with_fake_data)
   else:
     return DATASETS[name]()
+
```

Comment:
nit - is "nitpicking", picking to small things. we use it apologize for nitpicking to style (giving Google code style requirements, like 2 lines between functions/classes, spaces etc)